Wingstop_Recovery_Computer_V1/Wingstop_Recovery_Computer_V1.ino

```diff
@@ -228,57 +274,85 @@ while(!detect_apogee()){
   
   //update altitude at frequency (hz)
   iterate_altitude();
+  log_data();//logs data to sd card
+}
+
+  t_apogee = t_current; //saves apogee time
+  if (logFile) {
+    logFile.print(F("APOGEE DETECTED\t")); //logs event
+    reopen_file();//saves and reopens file
+  }
+
+while(MACH_DELAY > t_current){ //runs until MACH_DELAY is reached
   
+  //update altitude at frequency (hz)
+  iterate_altitude();
   log_data();//logs data to sd card
 
 }
 
-  reopen_file();//saves and reopens file
+  if (logFile) {
```

Comment:
The following block of code is repeated and deserves its own function (maybe call it `logLine` with a parameter called `msg`.)

```
 if (logFile) {
  logFile.print(F("<SOME EVENT HERE>\t")); //logs event
  reopen_file();//saves and reopens file
  }
```

Reply:
Couldn't find a way to force the string to be written to program memory, so it was taking a lot of RAM (about 6%) to write it as a function and pass in the string. Maybe there is another way I am missing....